src/pages/security/components/_hero_component.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { BackgroundImage, Header } from 'components/elements'
+import { Container } from 'components/containers'
+import device from 'themes/device'
+import { localize } from 'components/localization'
+
+const StyledContainer = styled(Container)`
+    flex-direction: column;
+    height: 100%;
+    width: 60%;
```

Comment:
you can remove this, because there are already sets of width in the `Container` components.. 

Add the `padding` here to replace background height.

```suggestion
    padding: 160px 0;
```